### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by tvigon**
Card de navigation

### CODE REVIEW

**Review by lcastelloni — APPROVED** [review]
Great!

File: App.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  <Text style={{alignSelf: 'center', fontSize: 30}}>{text}</Text>
+);
+
+function HomeScreen({navigation}) {
+  const [value, setValue] = useState('0');
+  const [aux, setAux] = useState('');
+  const [error, setError] = useState('');
+  const [logArray, setLogArray] = useState(['']);
+
+  return (
```
Comment (lcastelloni) [inline App.js]: Move this component the components folder. The main component, in this case is App, should be the only one inside App.js file

**Review by lcastelloni — CHANGES_REQUESTED** [review]

File: App.js [hunk 2]
```diff
@@ -10,13 +10,32 @@ import React, {useState} from 'react';
...
 import CalcButtons from './components/CalcButtons';
 import styles from './styles';
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
 
-import {SafeAreaView, Text, View} from 'react-native';
+import {Button, SafeAreaView, ScrollView, Text, TouchableOpacity, View} from 'react-native';
 
-const App = () => {
+function HistoryScreen({route, navigation}) {
```
Comment (lucawidergy) [inline App.js]: no, we dont use funcion. Always we have arrow function!. Also this shoul be in other folder called screens and a folder for this screen inside

File: App.js [hunk 3]
```diff
@@ -10,13 +10,32 @@ import React, {useState} from 'react';
...
+  const {logValue} = route.params;
+  return (
+    <View style={{flex: 1}}>
```
Comment (lucawidergy) [inline App.js]: put the style always in a style folder. we dont use inline styles

File: App.js [hunk 4]
```diff
@@ -26,16 +45,30 @@ const App = () => {
...
       <View style={[styles.row, styles.allButtons]}>
-        <CalcButtons
-          values={['7', '8', '9', '4', '5', '6', '1', '2', '3', '.', '0', '=']}
-          selectedValue={value}
-          setSelectedValue={setValue}
-          selectedAux={aux}
-          setSelectedAux={setAux}
-          setError={setError}
-        />
+        <View style={[styles.row, styles.numButtons]}>
```
Comment (lucawidergy) [inline App.js]: i think you forgot update this

**Review by lucawidergy — CHANGES_REQUESTED** [review]

**Comment by tvigon** [comment]
Navigation done. Deleted branch

**Pull request closed without merging**